src/http/keys/entities/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './response';
-export * from './key-with-module-address';
+// export * from './key-with-module-address';
```

Comment:
can be deleted

Reply:
will fix here https://github.com/lidofinance/lido-keys-api/pull/158